dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -290,17 +421,40 @@ public long authorizedJobAbandonByInstance(long instanceId) throws NoSuchJobInst
 
     private long authorizedJobPurgeAbandonByInstance(long instanceId) throws NoSuchJobInstanceException, JobSecurityException {
```

Comment:
Why don't we combine with restart/stop.   Take the isInAnyBatchRole() check here and combine with the overalll flow (with comments), minus exc handling at the very end of restart/stop.

Reply:
done